src/components/key_pad_menu/_key_pad_menu_item.scss

```
@@ -126,3 +122,12 @@
     border-color: $euiColorPrimaryText;
   }
 }
+
+.euiKeyPadMenuListItem {
```

To match our BEM style, lead with component name then the descriptive bit,

```suggestion
.euiKeyPadMenu__listItem {
```

This should also move to __key_pad_menu.scss_ to stay "next" to the component using the class.